src/inject/fallback.ts

```diff
@@ -1,3 +1,5 @@
+import {removeNode} from './utils/dom';
```

Comment:
Not sure, but I think this import will make the fallback bundle grow. Please check if so.